contrib/pg_tde/documentation/docs/how-to/enforcement.md

```diff
@@ -0,0 +1,14 @@
+# Encryption Enforcement
+
+For `pg_tde`, encryption enforcement ensures that only encrypted storage is allowed for specific operations, tables, or the entire database. It prevents the accidental creation of unencrypted tables or indexes in environments where encryption is required for compliance, security, or policy enforcement.
+
+## Enforcing encryption usage
+
+Use the following techniques to enforce the secure use of `pg_tde`.
```

Comment:
From use the following techniques, I would expect some examples or use cases to follow - but we have neither here, just a repeated description.